src/components/comment-card/comment-card.jsx

```diff
@@ -0,0 +1,19 @@
+import "./comment-card.css"
+
+function CommentCard(props){
+  const {comment} = props
+  console.log(comment)
+
+  return(
+    <>
```

Comment:
since the section tag wraps the everything else we don't really need to use a react fragment here.